apisix/core/schema.lua

```diff
@@ -20,7 +20,11 @@ local lrucache = require("apisix.core.lrucache")
 local cached_validator = lrucache.new({count = 1000, ttl = 0})
 local pcall = pcall
 
-local _M = {version = 0.3}
+local _M = {
+    version = 0.3,
+    TYPE_ANY = 1,
```

Comment:
- `TYPE_ANY` is confused, I think `TYPE_CONF` or `TYPE_OPT` would be better, how do you think?
- It seems like the code of checking plugin's config does not using this parameter, are there any special considerations?